config/ironic_ci.env

```diff
@@ -0,0 +1,11 @@
+HTTP_PORT=6180
```

Comment:
maybe that file can go under hack folder instead ?

Reply:
Yep, done